.eslintrc

```diff
@@ -219,6 +219,8 @@
     "vars-on-top": 0,
     "wrap-iife": 2,
     "wrap-regex": 0,
-    "yoda": [2, "never", {"exceptRange": true}]
+    "yoda": [2, "never", {"exceptRange": true}],
+
```

Comment:
extra line break intended here?

Reply:
Yeah, I wanted to separate the custom rules. Would have put in a comment to make it clear but this is faux-JSON so there are no comments... (We do use a blank line as a separator higher up in the file, at least.)